src/SmartComponents/CompletedTaskDetails/TaskEntries.js

```diff
@@ -8,35 +8,40 @@ import React from 'react';
 const converttorhelconversionSeverityMap = {
   error: {
     text: 'Inhibitor',
-    icon: <ExclamationCircleIcon color="#C9190B" />,
+    icon: <ExclamationCircleIcon />,
```

Comment:
It looks color prop has been removed (which is correct), but we need to make sure the icons have the same color after the migration. I am not able to test this. Because, I am not able to find where they are shown in the UI. Just raising for awareness.

Reply:
The color is now tracked via the `iconSeverityColor` value in this same object: https://github.com/RedHatInsights/tasks-frontend/pull/137/files/c0f0052c04dc712be5c50df16e56d61768a5dc0b#diff-7d2ef7c75d9bb6f0b5f8aadee16b935b906d47ad9d1d261ecce4ccf5922a2da8R12